src/main/java/org/symphonyoss/symphony/messageml/bi/BiContext.java

```diff
@@ -1,10 +1,12 @@
 package org.symphonyoss.symphony.messageml.bi;
 
+import joptsimple.internal.Strings;
```

Comment:
This import looks suspicious

Reply:
Nice catch, I wanted to import StringUtils instead. It will be fixed in next commit.